.github/workflows/_python-tests.yml

```diff
@@ -235,18 +203,9 @@ jobs:
         uses: PyO3/maturin-action@v1
         with:
           command: build
-          sccache: true
```

Comment:
Would we want to keep `sccache` here since we're not caching the build tools for this workflow?